.github/workflows/build-image-stable.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       matrix:
         brand_name: ["bluefin"]
     with:
-#     kernel_pin: 6.13.8-200.fc41.x86_64 ## This is where kernels get pinned.
+      kernel_pin: 6.14.5-200.fc41.x86_64 ## This is where kernels get pinned.
```

Comment:
Think this will need to be a F42 version.  Rolling back from 42 to 41 isn't something we can (easily) do.